erpnext/manufacturing/doctype/bom/bom.py

```
@@ -1854,7 +1865,13 @@ def item_query(doctype, txt, searchfield, start, page_len, filters):
 
 
 @frappe.whitelist()
-def make_variant_bom(source_name, bom_no, item, variant_items, target_doc=None):
+def make_variant_bom(
+	source_name: str,
+	bom_no: str,
```

Ruff (0.15.1)

[warning] Unused function argument: `bom_no`

(ARG001)